package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.6.1",
```

Comment:
- [ ] [REQUIRED] `redux` should be added to `package.json` [screenshot from Pathwright - redux and react-redux](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone1/AddReduxAndReact-Redux.png)